ibm/mas_devops/roles/db2/README.md

```diff
@@ -181,6 +181,17 @@ Define the Kubernetes memory limit for the Db2 pod.
 - Environment Variable: `DB2_MEMORY_LIMITS`
 - Default: `12Gi`
 
+### [optional] db2_ldap_username
```

Comment:
Please follow the same convention used in the other variables in the Readme ... e.g for the first one it should be:

```
### db2_ldap_username
Define the username of db2 in the local LDAP registry. If this defined, LDAP user will be the Manage user used to configure the JDBC configuration on MAS.

- Optional
- Environment Variable: `DB2_LDAP_USERNAME`
- Default: None
```